scripts/rmc.py

```diff
@@ -102,11 +102,11 @@ def run_cbmc(cbmc_filename, cbmc_args, verbose=False, quiet=False):
     cbmc_cmd = ["cbmc"] + cbmc_args + [cbmc_filename]
     return run_cmd(cbmc_cmd, label="cbmc", output_to="stdout", verbose=verbose, quiet=quiet)
 
-def run_visualize(cbmc_filename, cbmc_args, verbose=False, quiet=False, keep_temps=False, function="main", srcdir=".", wkdir="."):
-    results_filename = "results.xml"
-    coverage_filename = "coverage.xml"
-    property_filename = "property.xml"
-    temp_goto_filename = "temp.goto"
+def run_visualize(cbmc_filename, cbmc_args, verbose=False, quiet=False, keep_temps=False, function="main", srcdir=".", wkdir=".", outdir="."):
+    results_filename = os.path.join(outdir, "results.xml")
+    coverage_filename = os.path.join(outdir, "coverage.xml")
+    property_filename = os.path.join(outdir, "property.xml")
+    temp_goto_filename = os.path.join(outdir, "temp.goto")
```

Comment:
We should involve @vecchiot-aws in this PR to check at least edits to this file.